components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -200,6 +214,14 @@ open class BrowserMenu internal constructor(
         menuList?.let { adapter.invalidate(it) }
     }
 
+    private fun setColors(menuLayout: View, colorState: MenuStyle?) {
+        val listParent: CardView = menuLayout.findViewById<CardView>(R.id.mozac_browser_menu_menuView)
+        backgroundColor = colorState?.backgroundColor?.let {
+            listParent.setCardBackgroundColor(it)
+            it.defaultColor
+        } ?: listParent.cardBackgroundColor.defaultColor
```

Comment:
Why do we need this `.let`? Can we just do this: 
```suggestion
        backgroundColor = listParent.cardBackgroundColor.defaultColor
```

Reply:
The `let` block is where we are also setting the background for the menu to be the one from the MenuStyle.
It's basically this code https://github.com/mozilla-mobile/android-components/blob/765ebf9829eb3622dddef964179ef285c2a7fc16/components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt#L70-L72 refacored.